subworkflows/local/setup_reference_data/main.nf

```diff
@@ -44,12 +44,12 @@ workflow SETUP_REFERENCE_DATA {
 
     //
     // Create reference meta-map for downstream steps
-    //
+    //  The irida_id key is there to not error out when creating the upload file with nf-iridanext
```

Comment:
Could you describe the specific error you encounter that requires adding a new `irida_id`?

Reply:
Ah, okay. That makes sense. Thanks for the explanation.